src/System.Management.Automation/security/nativeMethods.cs

```diff
@@ -86,6 +86,9 @@ internal partial class NativeConstants
 
         /// MAX_PATH -> 260
         internal const int MAX_PATH = 260;
+
+        /// This function is not supported on this system
```

Comment:
:-) CodeFactor says that we use incorrect documentation header.

Reply:
I'll fix this file